src/SpectralUnmixing.jl

```diff
@@ -311,49 +505,78 @@ function unmix_line(line::Int64, reflectance_file::String, mode::String, refl_no
     end
     return line, mixture_results, good_data, mixture_results_std, complete_fractions
-
 end
 
-function unmix_and_write_line(line::Int64, reflectance_file::String, mode::String, refl_nodata::Float64,
-                    refl_scale::Float64, normalization::String, library::SpectralLibrary, output_files::Vector{String},
-                    write_complete_fractions::Bool,
-                    reflectance_uncertainty_file::String = "", n_mc::Int64 = 1,
-                    combination_type::String = "all", num_endmembers::Vector{Int64} = [2,3],
-                    max_combinations::Int64 = -1, optimization="bvls")
-    
-    line_results = unmix_line(line, reflectance_file, mode, refl_nodata, refl_scale, normalization, library, 
+"""
+    unmix_and_write_line(line::Int64, reflectance_file::String, mode::String,
+                          refl_nodata::Float64, refl_scale::Float64,
+                          normalization::String, library::SpectralLibrary,
+                          output_files::Vector{String}, write_complete_fractions::Bool,
+                          reflectance_uncertainty_file::String="", n_mc::Int64=1,
+                          combination_type::String="all",
+                          num_endmembers::Vector{Int64}=[2, 3],
+                          max_combinations::Int64=-1, optimization="bvls")
+
+Unmix the specified `line` of `reflectance_file` and write the results to `output_files`.
+
+- Calls [`unmix_line`(@ref)] and [`write_line_results`](@ref), see for arguments.
+"""
+function unmix_and_write_line(line::Int64, reflectance_file::String, mode::String,
+    refl_nodata::Float64, refl_scale::Float64, normalization::String,
+    library::SpectralLibrary, output_files::Vector{String}, write_complete_fractions::Bool,
+    reflectance_uncertainty_file::String="", n_mc::Int64=1, combination_type::String="all",
+    num_endmembers::Vector{Int64}=[2, 3], max_combinations::Int64=-1, optimization="bvls")
+
+    line_results = unmix_line(
+        line, reflectance_file, mode, refl_nodata, refl_scale, normalization, library,
         reflectance_uncertainty_file, n_mc, combination_type, num_endmembers,
-        max_combinations, optimization)
+        max_combinations, optimization
+    )
 
-    write_line_results(output_files, line_results, n_mc, write_complete_fractions) 
-    
+    write_line_results(output_files, line_results, n_mc, write_complete_fractions)
 end
 
+"""
+    class_assign_fractions(complete_fractions, library::SpectralLibrary)
+
+Aggregate endmember fractions into per-unique-class fractions.
+
+# Arguments
+- `complete_fractions`: A matrix or vector containing fractions for each pixel,
+where each column represents an endmember.
+- `library::SpectralLibrary`: The spectral library of the endemembers used in unmixing.
+
+# Returns
+- A matrix or vector of class fractions, where each row corresponds to a pixel and each
+column corresponds to a unique class from the library.
+"""
```

Comment:
This was previously used, and has been since out-dated.  We can safely remove, and always add back in if needed - good catch.  Please resolve once removed.

Reply:
Removed.